bin/version.sh

```diff
@@ -1,4 +1,3 @@
 set -e
-./node_modules/.bin/webpack
-./node_modules/.bin/webpack --minify
-npm run build
+yarn run dist
+yarn run build
```

Comment:
let's remove this .sh file and just have this as in package.json scrips
like

Reply:
SGTM